.github/workflows/pr-builder.yml

```
@@ -17,9 +17,12 @@ jobs:
   build:
     runs-on: ubuntu-latest
 
+    env:
+      JAVA_TOOL_OPTIONS: "-Djdk.util.zip.disableZip64ExtraFieldValidation=true -Djdk.nio.zipfs.allowDotZipEntry=true"
+
     strategy:
       matrix:
-        java-version: [ 11.0.19+7, 17.0.7+7 ]
+        java-version: [ 11.0.20+8, 17.0.7+7 ]
```

we should be able to set the major version here and the action would fetch the latest
```suggestion
        java-version: [ 11., 17 ]
```